proofs/tacmach.mli

```diff
@@ -139,4 +139,6 @@ module New : sig
 
   val pf_nf_evar : Proofview.Goal.t -> constr -> constr
 
+  (** Gathers the undefined evars of the current goal. *)
```

Comment:
Sorry to nitpick on this detail, but given the questions of @Zimmi48 ...
It is not the current goal, but rather the goal given as input.